var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -210,6 +213,11 @@ def install(self, spec, prefix):
             '--with-blas-lapack-lib=%s' % lapack_blas.joined()
         ])
 
+        if 'x' in spec:
```

Comment:
I'm not 100% sure that this works. I would use this instead:
```suggestion
        if '+x' in spec:
```